app/templates/components/widgets/forms/ticket-input.hbs

```diff
@@ -24,14 +24,14 @@
     <div class="field">
       <div class="ui icon buttons">
         <button type="button" class="ui button" {{action 'toggleSettings'}} data-content="Settings"><i class="setting icon"></i></button>
-        <button type="button" class="ui button" {{action removeTicket}} data-content="Delete"><i class="trash icon"></i></button>
+        <button type="button" class="ui button" {{action 'removeTicket'}} data-content="Delete"><i class="trash icon"></i></button>
```

Comment:
Revert this. These are all actions passed in. Please don't make such changes blindly without checking the code.